src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -418,95 +387,94 @@ class MoneyRequestAmountPage extends React.Component {
-                                style={[styles.w100, styles.justifyContentEnd, styles.pageWrapper]}
-                                nativeID={this.numPadContainerViewID}
-                            >
-                                {DeviceCapabilities.canUseTouchScreen() ? (
-                                    <BigNumberPad
-                                        nativeID={this.numPadViewID}
-                                        numberPressed={this.updateAmountNumberPad}
-                                        longPressHandlerStateChanged={this.updateLongPressHandlerState}
-                                    />
-                                ) : (
-                                    <View />
-                                )}
-
-                                <Button
-                                    success
-                                    style={[styles.w100, styles.mt5]}
-                                    onPress={this.navigateToNextPage}
-                                    pressOnEnter
-                                    isDisabled={!this.state.amount.length || parseFloat(this.state.amount) < 0.01}
-                                    text={buttonText}
+                        </View>
+                        <View
+                            onMouseDown={(event) => onMouseDown(event, [numPadContainerViewID, numPadViewID])}
+                            style={[styles.w100, styles.justifyContentEnd, styles.pageWrapper]}
+                            nativeID={numPadContainerViewID}
+                        >
+                            {DeviceCapabilities.canUseTouchScreen() ? (
+                                <BigNumberPad
+                                    nativeID={numPadViewID}
+                                    numberPressed={updateAmountNumberPad}
+                                    longPressHandlerStateChanged={updateLongPressHandlerState}
                                 />
-                            </View>
+                            ) : (
+                                <View />
+                            )}
+
+                            <Button
+                                success
+                                style={[styles.w100, styles.mt5]}
+                                onPress={navigateToNextPage}
+                                pressOnEnter
+                                isDisabled={!amount.length || parseFloat(amount) < 0.01}
+                                text={buttonText}
+                            />
                         </View>
-                    )}
-                </ScreenWrapper>
-            </FullPageNotFoundView>
-        );
-    }
+                    </View>
+                )}
+            </ScreenWrapper>
+        </FullPageNotFoundView>
+    );
 }
 
 MoneyRequestAmountPage.propTypes = propTypes;
 MoneyRequestAmountPage.defaultProps = defaultProps;
+MoneyRequestAmountPage.displayName = 'MoneyRequestAmountPage';
 
 export default compose(
```

Comment:
Sorry for late suggestion, just remembered this, replace every HOC below with it's hook equivalent (except for withOnyx, it's still WIP https://github.com/Expensify/react-native-onyx/pull/244)